cli/src/command/account.rs

```diff
@@ -938,25 +965,26 @@ pub async fn voting_output_command(account: &Account) -> Result<(), Error> {
     Ok(())
 }
 
-async fn print_address(account: &Account, address: &AccountAddress) -> Result<(), Error> {
+// TODO: display AccountAddress infos (key_index, change_address, etc) in the `addresses` command generated list?
```

Comment:
Want to do in this PR?
If we want this information, I think it should just print the bip44 values as array to not bloat the output

Reply:
I agree ... yeah ... fine doing it in this PR :+1:

Reply:
I guess it doesn't make sense anymore with NFT and Alias addresses also listed there now, so let's resolve?

Reply:
Well for Alias and NFT it would be just not existent, still interesting for the others
But I'm also fine not having it in this PR or at all, but then remove the TODO or open an issue :trollface:

Reply:
chose to remove the TODO